pyproject.toml

```diff
@@ -78,8 +87,17 @@ exclude = [
     "venv",
     "_version.py"
 ]
+target-version = "py311"
 
-lint.ignore = [
+[tool.ruff.lint.per-file-ignores]
+"**/test{s,}/*.py" = [
+    "S101",  # https://docs.astral.sh/ruff/rules/assert/
+    "D",  # https://docs.astral.sh/ruff/rules/#pydocstyle-d
+    "ARG001",  # https://docs.astral.sh/ruff/rules/unused-function-argument/
+    "INP",  # https://docs.astral.sh/ruff/rules/#flake8-no-pep420-inp
 ]
 
-target-version = "py311"
+[tool.ruff.lint.flake8-quotes]
+docstring-quotes = "double"  # https://docs.astral.sh/ruff/settings/#lint_flake8-quotes_docstring-quotes
+inline-quotes = "single"  # https://docs.astral.sh/ruff/settings/#lint_flake8-quotes_inline-quotes
```

Comment:
I'm really not a fan of this (unless I've misunderstood it). I've stuck with the C styling where strings get double and chars get single.

Reply:
I'm also just not a fan of single quotes period lol

Reply:
Also, 🙏 please don't make me have to change 😭